packages/replay-next/pages/variables.css

```diff
@@ -308,8 +310,8 @@
   --point-panel-timeline-label-background-color: #000000;
   --point-panel-timeline-label-color: var(--color-brand);
 
-  --badge-picker-background-color: #28282a;
-  --badge-default-color: #747476;
+  --badge-picker-background-color: rgba(0, 0, 0, 0.04);
+  --badge-default-color: rgba(0, 0, 0, 0.08);
```

Comment:
cc @jonbell-lot23 These changes roughly approximate the colors we had before, but also handle error states without requiring _additional_ variables to track. We can iterate on this if needed though.

Reply:
Hm, disregard. I think I'm going to have to add new variables for these invalid states– because of this 💩
<img width="141" alt="Screen Shot 2023-01-19 at 9 48 50 AM" src="https://user-images.githubusercontent.com/29597/213473255-03df9c0e-6468-4d55-a100-075d1e88f3e9.png">